pipelined/bevy_render2/src/texture/image.rs

```diff
@@ -25,15 +25,17 @@ pub struct Image {
 
 impl Default for Image {
     fn default() -> Self {
+        let format = wgpu::TextureFormat::Rgba8UnormSrgb;
```

Comment:
Should this be `TextureFormat::bevy_default()`? It would stay `Rgba8UnormSrgb` on android and otherwise be`Bgra8UnormSrgb`.

Reply:
I changed it to `bevy_default()`